sse2neon.h

```diff
@@ -5390,31 +5390,21 @@ FORCE_INLINE __m128i _mm_sll_epi64(__m128i a, __m128i count)
 //   r7 := a7 << count
 //
 // https://msdn.microsoft.com/en-us/library/es73bcsy(v=vs.90).aspx
```

Comment:
Change to Intel Intrinsics website hyperlink.

Reply:
Done.